src/NUnitConsole/nunit3-console/ConsoleRunner.cs

```diff
@@ -310,6 +325,14 @@ private static string GetOSVersion()
 
         private void DisplayExtensionList()
         {
+            if (_options.ExtensionDirectories.Count > 0)
+            {
+                _outWriter.WriteLine(ColorStyle.SectionHeader, "User Extension Directories");
+                foreach (var dir in _options.ExtensionDirectories)
+                    _outWriter.WriteLine($"  {Path.GetFullPath(dir)}");
+                _outWriter.WriteLine();
+            }
+
             _outWriter.WriteLine(ColorStyle.SectionHeader, "Installed Extensions");
 
             foreach (var ep in _extensionService?.ExtensionPoints ?? new IExtensionPoint[0])
```

Comment:
Although it's not part of the PR, I do find it surprising for a different reason... There are always extension points, although there may not be extensions! I believe this goes way back to when we first developed the extensibility infrastructure but had not added any extension points. At the present time this would be some kind of internal error, so I think I should just throw.

Reply:
I now throw in  the class constructor if there are no extension points.